CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 Here we document changes that affect the public API or changes that needs to be communicated to other developers. 
 
+## 2022-09-29 Pre-compiled headers overhaul
+The generation of pre-compiled headers (PCH) was changed from Cotire to CMake. This enables using PCH in combination with Qt6.
+A single PCH file is generated for inviwo core including a number of standard includes. In addition, there is a PCH file for the qtwidgets modules with some Qt headers. These PCH files ar reused by all modules by default when depending on qtwidgets or core respectively, and assuming `IVW_CFG_PRECOMPILED_HEADERS` is checked. Use the `NO_PCH` option when calling `ivw_create_module()` to prevent the default PCH from being used in a module.
```

Comment:
Not sure if this need updating, might explain the difference between using target_precompile_headers directly and ivw_compile_optimize_on_target

Reply:
good suggestion